### PR TITLE
add inner error to errors on send 6300

### PR DESCRIPTION
**Pull request by luu-alex**
## Description
#6300 
Error objects that are returned from an RPC is not included in the error, making it an issue if a user wanted to keep track of this. For example when cancelling a send transaction on metamask, metamask will send back an error and code but we currently do not include that in the error.  This PR adds the error Object to be apart of the innerError property. 
Please include a summary of the changes and be sure to follow our [Contribution Guidelines](../CONTRIBUTIONS.md).

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] I have selected the correct base branch.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] I ran `npm run lint` with success and extended the tests and types if necessary.
- [ ] I ran `npm run test:unit` with success.
- [ ] I ran `npm run test:coverage` and my test cases cover all the lines and branches of the added code.
- [ ] I ran `npm run build` and tested `dist/web3.min.js` in a browser.
- [ ] I have tested my code on the live network.
- [ ] I have checked the Deploy Preview and it looks correct.
- [ ] I have updated the `CHANGELOG.md` file in the root folder.
- [ ] I have linked Issue(s) with this PR in "Linked Issues" menu.

### CODE REVIEW

**Review by avkos — APPROVED** [review]

**Comment by avkos** [comment]
LGTM, please add test coverage

**Comment by luu-alex** [comment]
@avkos  fixed, thanks

File: packages/web3-core/CHANGELOG.md [hunk 1]
```diff
@@ -158,6 +158,7 @@ Documentation:
 ### Fixed
 
 -   Fixed the issue: "Version 4.x does not fire connected event for subscriptions. #6252". (#6262)
+-   Fixed rpc errors not being sent as an inner error when using the `send` method on request manager (#6300).
```
Comment (jdevcs) [inline packages/web3-core/CHANGELOG.md]: this needs update, as we have now 4.1.0

Comment (jdevcs) [reply packages/web3-core/CHANGELOG.md]: apart from that lgtm

Comment (luu-alex) [reply packages/web3-core/CHANGELOG.md]: updated. thank you

**Review by jdevcs — APPROVED** [review]

**Pull request merged**